istio/istio-egress.yml

```diff
@@ -16,6 +16,9 @@ data:
     - kind: io.l5d.recentRequests
       sampleRate: 0.25
 
+    usage:
+      orgId: linkerd-examples-istio-egress
```

Comment:
I mentioned this before, but since this acts as our official istio distro it weirds me out to put "linkerd-examples-xxx" as the orgId. Either omit altogether, or maybe change the name to "linkerd-istio-egress'